releasenotes/notes/elrepo-9-9d88af1dea541155.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Provide ELRepo 9, which in turn provides packages to support be2net
+    and mpt3sas hardware.
```

Comment:
Mention that it's disabled by default and how to enable it?

Reply:
Added